.github/workflows/pr.yml

```diff
@@ -32,33 +34,13 @@ jobs:
         java: [8, 21]
```

Comment:
in the https://github.com/apache/kafka/pull/16936#issuecomment-2299395186, you say PR will run compile check for all JDKs